CppCoreGuidelines.md

```diff
@@ -19629,9 +19629,18 @@ Failing to follow this results in difficult to diagnose errors due to picking up
 
 Library creators should put their headers in a folder and have clients include those files using the relative path `#include <some_library/common.h>`
 
+##### Project Example
+
+Consider a project with modules where src/module2/foo.cpp is using src/module1/utilities.hpp:
```

Comment:
I don't think you're referring to the C++ language feature 'modules', so this is a poor choice of wording.

Reply:
Correct. I updated it to use the more generic term component (a modular, self-contained unit of software that encapsulates specific functionality). Components may be built into .so/.dll's and linked or loaded by the main executable. Alternatively, all sources of multiple components can be compiled into one executable. Should I a definition of component to be super clear, or is the term component enough?